Configuration/Geometry/python/dict2023Geometry.py

```diff
@@ -1104,6 +1104,7 @@
     ("O2","T14","C3","M2","F2","I1") : "D40",
     ("O3","T14","C8","M3","F2","I9") : "D41",
     ("O2","T15","C3","M2","F2","I1") : "D42",
+    ("O2","T14","C4","M2","F2","I7") : "D43",
```

Comment:
is it intentional to use older muon and MTD versions here? This makes it not an exact copy of D41.

Reply:
The only difference was in the RPC specpars. Now I have made it M3

Reply:
Ah, and now I recall that I9 and I7 are the same except for the HGCal positioning.

Reply:
@kpedro88 @bsunanda I9 and I7 are identical as far as BTL is concerned, but ETL is moved and "hammered" (i.e. less active modules) to fit in the new mother volume in D41.  So HGCal v9 cannot use MTD I9

Reply:
@fabiocos thanks for confirming